source/features/cleanup-repo-filelist-actions.tsx

```diff
@@ -14,16 +13,15 @@ function init(): void {
 	searchButton.firstChild!.replaceWith(<SearchIcon/>);
 
 	const addButtonWrapper = searchButton.nextElementSibling!;
-	const addButton = select('.btn', addButtonWrapper);
-	if (addButton) {
-		addButton.classList.add('d-md-block', 'tooltipped', 'tooltipped-ne');
-		addButton.classList.remove('d-md-flex', 'ml-2');
-		addButton.setAttribute('aria-label', 'Add file');
-		addButton.textContent = '';
-		addButton.append(<PlusIcon/>);
+	if (addButtonWrapper.nodeName === 'DETAILS') {
+		addButtonWrapper.classList.add('tooltipped', 'tooltipped-ne');
+		addButtonWrapper.setAttribute('aria-label', 'Add file');
 
-		searchButton.classList.remove('mr-2');
-		groupButtons([searchButton, addButtonWrapper]);
+		const addIcon = select('.btn span', addButtonWrapper)!;
+		addIcon.classList.remove('d-md-flex');
+		addIcon.classList.add('d-md-block');
+		addIcon.textContent = '';
+		addIcon.append(<PlusIcon/>);
```

Comment:
I think this can use the same code we use for the `searchButton`
```suggestion
		addIcon.firstChild!.replaceWith(<PlusIcon/>);
```